src/CustomBigQueryClient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\BigQuery;
+
+use Google\Cloud\BigQuery\BigQueryClient;
+use Google\Cloud\BigQuery\JobConfigurationInterface;
+use Google\Cloud\BigQuery\QueryJobConfiguration;
+use Google\Cloud\BigQuery\QueryResults;
+
+class CustomBigQueryClient extends BigQueryClient
```

Comment:
`BigQueryClientWrapper` bude imho lepší ty neděláš custom clienta

Reply:
Jo to asi jo dik :)